### PR TITLE
adding the config to the mail and blades to the main actions in the app

### PR DESCRIPTION
**Pull request by alejandroGonGon**
Adding the main configuration to send Mails in the main actions like create a : User, Petition, Donation. 
Also adding the templates for the mails.

Demo: 
image
image
image
image
image
image

### CODE REVIEW

**Comment by LautaroPardo0210** [comment]
Task linked: [CU-1tdrx29 Update mailer config.](https://app.clickup.com/t/1tdrx29)

File: app/Mail/UserRegister.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+{
+    use Queueable, SerializesModels;
+
+    /**
+     * Created vars to encapsulate data like subject, name, email etc
+     * @return void
+    */
+    public $userEmail;
+
+    public $subject = "Usuario creado exitosamente";
```
Comment (nicocadq) [inline app/Mail/UserRegister.php:20]:
```suggestion
    public $subject = "Te has registrado exitosamente";
```

File: config/mail.php [hunk 2]
```diff
@@ -84,8 +84,8 @@
     */
 
     'from' => [
-        'address' => env('MAIL_FROM_ADDRESS', 'hello@example.com'),
-        'name' => env('MAIL_FROM_NAME', 'Example'),
+        'address' => env('MAIL_FROM_ADDRESS', 'hoydonamos@gmail.com'),
+        'name' => env('MAIL_FROM_NAME', 'Che, ¿Hoy donamos?'),
```
Comment (nicocadq) [inline config/mail.php:88]: Does it are added on an env file? And this date (the second param of the `env()`) is loaded on fallback if the env is not found?

File: resources/views/emails/donationCreated.blade.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Donacion registrada</title>
+</head>
+<body>
+
+    <h1>Has creado una Dpnacion exitosamente!</h1>
```
Comment (nicocadq) [inline resources/views/emails/donationCreated.blade.php:11]: fix typos

File: resources/views/emails/donationCreated.blade.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Donacion registrada</title>
```
Comment (nicocadq) [inline resources/views/emails/donationCreated.blade.php:7]:
```suggestion
    <title>Donación</title>
```

**Review by nicocadq — CHANGES_REQUESTED** [review]
Add the text to a locales file, try to take advantage of the internationalization features of Laravel. Also, there are a lot of typos on the email's body, try to put an eye on that it does not look well. 

Nice to make it work! We can tackle the design latter.

File: config/mail.php (hunk 2 above)
Comment (alejandroGonGon) [reply config/mail.php:88]: These already exits in the env file, and yes, if it doesnt work in the env dile would use the second param. I added it beacuse the doc of mail ask to do it.

File: resources/views/emails/donationCreated.blade.php (hunk 3 above)
Comment (alejandroGonGon) [reply resources/views/emails/donationCreated.blade.php:11]: Je

Comment (alejandroGonGon) [reply resources/views/emails/donationCreated.blade.php]: Done!

**Comment by alejandroGonGon** [comment]
I run a command that will always ruin the files and will bring conflicts into the future, so this PR will be replaced by other without conflicts or anything that breaks the entire code, as this does.

**Comment by alejandroGonGon** [comment]
I run a command that will always ruin the files and will bring conflicts into the future, so this PR will be replaced by other without conflicts or anything that breaks the entire code, as this does.

**Comment by alejandroGonGon** [comment]
> I run a command that will always ruin the files and will bring conflicts into the future, so this PR will be replaced by other without conflicts or anything that breaks the entire code, as this does.

I confused with another branch, so thats why I closed it.

**Review by nicocadq — APPROVED** [review]

**Pull request merged**